controllers/articlesController.js

```diff
@@ -0,0 +1,19 @@
+const getArticleFromDB = require('../models/articlesModels')
+
+function getParticularArticle(req, res, next) {
+
+    const { article_id } = req.params
+
+    getArticleFromDB(article_id).then((article) => {
+
+        res.status(200).send(article)
```

Comment:
When sending a response, make sure to send it in an object
eg
`res.status(200).send({article})`
This ensures uniformity and consistency across responses, and allows other fields to easily be added if needed